### ISSUE TITLE
Nested sourcecode within examples not rendering correctly in Word

### ISSUE THREAD

**Issue by opoudjis** [metanorma/metanorma-iso#1256]
From https://github.com/ISOTC171SC2/32000-2-2020-amd1/issues/1

![image](https://github.com/user-attachments/assets/22fda679-f208-4fec-8786-bd5a8da86e1a)

> The source blocks are just [source] and look perfect in both the PDF and HTML outputs - just not in DOC. It appears hard line breaks are not being created in the DOC output (the spaces correctly come through as nbsp).

> The problem with the code snippets looking wrong in Word, appears to be a
problem with the use of source blocks inside of an example block.  If you
put them in top level source blocks - it looks fine in the Doc as well.

**Comment by opoudjis** [comment]
> Easiest fix for source code blocks is just to change to left-justify - MN output is fully justified (which is bad for a source block!).

It's likely that left-justify is being applied to sourcecode but not example/sourcecode in the Word CSS

**Comment by opoudjis** [comment]
This renders correctly in DIS (stage = 60), but not in default Word styling (stage = 30, as in the given document).

(@petervwyatt one more reason why I need access to the source documents to work out what is going on, which I now have. The stage of the document affects its rendering in Word, and I wouldn't have realised that was the issue.)

In the default Word styling, the styles Sourcecode and pseudocode are marked as left-aligned; Code, which is what this paragraph is getting, isn't so marked, so it is ending up as justify by default. Code not Sourcecode is what ends up used, and I think that's just an oversight when I was translating the source code of CSS. Making Code left-aligned.

**Comment by opoudjis** [comment]
<img width="972" alt="Image" src="https://github.com/user-attachments/assets/468700ae-1f8d-4d6c-a19f-641bda48fa1e" />

**Comment by opoudjis** [comment]
Yes, but that paragraph is justified, and it is supposed to be as a normal paragraph. Whenever you put a line break in a paragraph, you must change it to alignment to left-aligned: Word is consistent in that regard: justified right margins in Word ignore line breaks, and I am not convinced that behavior is incorrect.

It is up to the editors of the document to work out whether the line break is supposed to be there, in which case put [align=left] on the paragraph, or whether as I suspect it has been inserted in error, in which case it should be deleted.

**Comment by petervwyatt** [comment]
Everything is working for me now both via Gems and latest Docker.
